paymentsheet/src/main/java/com/stripe/android/paymentsheet/verticalmode/ManageScreenInteractor.kt

```diff
@@ -98,9 +98,7 @@ internal class DefaultManageScreenInteractor(
     init {
         coroutineScope.launch {
             state.collect { state ->
-                if (!state.canEdit) {
-                    safeNavigateBack()
-                } else if (!state.isEditing && !state.canRemove && state.paymentMethods.size == 1) {
+                if (!state.isEditing && !state.canEdit && state.paymentMethods.size == 1) {
```

Comment:
Had to remove this behavior for vertical mode since it's now possible to have multiple payment methods but be able to remove none of them.

Reply:
if you have multiple non-modifiable payment methods and you can't remove them, wouldn't state.canEdit be false? I'm a bit confused about how that removes the need for this branch

Reply:
If we have multiple non-removable, non-editable payment methods, I imagine we still want to allow customers to switch their payment selection correct?

Reply:
ahh I see